duracloud/management-console
# Add Swift S3 as a storage provider option

**JIRA Ticket**: (link)

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?

This PR adds Swift S3 as a storage option in the root console when selecting storage providers. When configured, the user will be asked for an access key, secret key, S3 endpoint and signer type.

# How should this be tested?

To be fully tested, this PR requires an associated PR to the base Duracloud repo. But to test just this, one can:
* Create a new account with Swift S3 as the primary storage provider.
* Configure the provider .
* Check that the configuration has made it to the `storage_provider_account_properties` table in the `duracloud_accounts` database.

# Additional Notes:

Documentation will need to be updated with details on how to use the new endpoint. (I will be writing this).

# Interested parties
@bbranan
